### ISSUE TITLE
Tooltip hotkey identifier

### ISSUE THREAD

**Issue by thorsummoner** [linuxmint/nemo#1040]
As an end user
I would like clickable items that have tooltips, that also have hotkeys, to expose the hoykey in the tooltip.

This is a first order of accessibility thing to me, I want to do an action with a hotkey, read the ui element's tooltup. This is probably something that should be proposed to the gnome UI/UX crowd but I found myself wanting it first here.

(I was able to find the hotkey I wanted by assumptive-correlation between the button I was clicking and menu bar item that seemed to match)

**Comment by icarter09** [comment]
Many thanks for submitting your feature request for Nemo. Your feature request has been documented and will be reviewed by the Linux Mint team.

For more information on our workflow and feature requests, read https://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/faq.html.
